### PR TITLE
Update desktop developers page

### LINKED ISSUE #2888: Update /desktop/developers - Build
Replace “Myapps” screenshot with dev.u.c - change to build.snapcraft.io
Update copy https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit
Update screenshot (x2 wallpapers)
Update graph
delete icons or images from the contextual footer

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update copy and images on `/desktop/developers`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/desktop/developers>
- Check that page matches [the design](https://user-images.githubusercontent.com/36884067/38744378-98a2bc3c-3f39-11e8-8aad-c7685a03382a.jpg) and [the copy doc](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit#)

## Issue / Card

Fixes #2888

### CODE REVIEW

**Comment by steverydz** [comment]
@Caleb-Ellis Addressed all your comments.

I embedded the svg icons so I could control the fill colour.

**Comment by anasereijo** [comment]
Hi @steverydz this is looking good just some tiny things:

**Graph section**
can you please remove the margin between both text lines? see bellow
screen shot 2018-04-17 at 14 51 18

**Tools Developers need section**
can you amend the padding between logos? move them closer like I have on the original design. You can make the right section 7 col if needed. see bellow
screen shot 2018-04-17 at 15 05 48

Thank you

**Comment by frankiegranato** [comment]
@steverydz & @anasereijo I noticed a little colour discrepancy on the third icon in the "Get started ..." section. The third icon (Build your first snap) looks black and the other two look grey. Lets make them all grey please?

**Comment by anasereijo** [comment]
@frankiegranato thanks for checking that. That was my mistake - sent the wrong link. @steverydz can you please replace that icon with this one: https://assets.ubuntu.com/v1/547ba157-manage-ubuntugrey.svg
I'm also going to update the copydoc

**Comment by steverydz** [comment]
@anasereijo I've updated the grid for the icons.

**Comment by anasereijo** [comment]
@steverydz it's looking much better! sorry to be a pain but can you make the spacing now slightly bigger between them (to look more like my final design)? around 40px. thanks

**Comment by frankiegranato** [comment]
Looks good to me @steverydz

**Comment by steverydz** [comment]
@anasereijo It might be simpler to have the logos grouped together as one image?

**Comment by anasereijo** [comment]
@steverydz here you go! https://assets.ubuntu.com/v1/24b49062-LOGOS.png

**Comment by steverydz** [comment]
@anasereijo Updated

File: templates/desktop/developers.html [hunk 1]
```diff
@@ -132,7 +184,7 @@ <h2>By developers, for developers</h2>
         <p><a class="p-link--external" href="https://certification.ubuntu.com/desktop">See all Ubuntu certified PCs</a></p>
       </div>
       <div class="col-5 u-vertically-center">
-        <img src="{{ ASSET_SERVER_URL }}7d0573d2-desktop-developer-developers.jpg" alt="Developer laptop" />
+        <img src="{{ ASSET_SERVER_URL }}89abf289-Dell_XPS_Laptop_Left-Developer.png" alt="Developer laptop" />
       </div>
     </div>
   </div>
```
Comment (anthonydillon) [inline templates/desktop/developers.html]: Don't need this closing div

Comment (steverydz) [reply templates/desktop/developers.html]: Good spot!

**Comment by steverydz** [comment]
@anthonydillon Fixed

**Pull request merged**